.github/workflows/acceptance.yaml

```diff
@@ -31,6 +34,14 @@ jobs:
           go get -u github.com/onsi/ginkgo/ginkgo
       - name: Setup Flux v2 CLI
         uses: fluxcd/flux2/action@main
+      - name: install libgit
```

Comment:
You have to BUILD it? :O

Reply:
The version available in 'apt' isn't correct and I couldn't find a mirror :(